components/measurement/nettests/WebConnectivity.js

```diff
@@ -167,32 +167,38 @@ FailureString.propTypes = {
   failure: PropTypes.string
 }
 
+const DnsNarrowAnswerCell = (props) => (
+  <Box width={1/12}>{props.children}</Box>
+)
+
 const DnsAnswerCell = (props) => (
-  <Box width={1/8}>{props.children}</Box>
+  <Box width={1/4}>{props.children}</Box>
 )
 
 DnsAnswerCell.propTypes = {
   children: PropTypes.any
 }
 
-const FiveColRow = ({ name = 'Name', netClass = 'Class', ttl = 'TTL', type = 'Type', data = 'DATA', header = false}) => (
+const DnsAnswerRow = ({ name = 'Name', netClass = 'Class', ttl = 'TTL', type = 'Type', data = 'DATA', anc = 'Ancillary info', header = false}) => (
   <Text fontWeight={header ? 'bold' : undefined}>
     <Flex flexWrap='wrap' mb={2}>
       <DnsAnswerCell>{name}</DnsAnswerCell>
-      <DnsAnswerCell>{netClass}</DnsAnswerCell>
-      <DnsAnswerCell>{ttl}</DnsAnswerCell>
-      <DnsAnswerCell>{type}</DnsAnswerCell>
+      <DnsNarrowAnswerCell>{netClass}</DnsNarrowAnswerCell>
+      <DnsNarrowAnswerCell>{ttl}</DnsNarrowAnswerCell>
+      <DnsNarrowAnswerCell>{type}</DnsNarrowAnswerCell>
       <DnsAnswerCell>{data}</DnsAnswerCell>
+      <DnsAnswerCell>{anc}</DnsAnswerCell>
     </Flex>
   </Text>
 )
 
-FiveColRow.propTypes = {
+DnsAnswerRow.propTypes = {
   name: PropTypes.string,
   netClass: PropTypes.string,
   ttl: PropTypes.number,
   type: PropTypes.string,
   data: PropTypes.string,
+  anc: PropTypes.string,
```

Comment:
Oh, it would be great if you called out non global IPs, which you can check with https://docs.python.org/3/library/ipaddress.html#ipaddress.IPv4Address.is_global

Perhaps even differentiate loopback, from private net, etc...

That will make it a lot more obvious to the user what the issue with the answer is.

In that case, perhaps the header should be "Answer IP Info"

Reply:
Thank you very much for your comments! I have renamed the column as you suggested.

Detecting private or loopback addresses is a great idea! And yes, Python has these very nice methods.  Alas, I don't know how to do this in Javascript.

Guessing again how the data has been constructed... for addresses not found in global routing, `asn` will be null.  This probably holds for all private (including loopback) addresses.  Right now the code doesn't output anything in this case.  So this case is (negatively) visible.  We could be more explicit by writing e.g. _non-routed address_ if the DNS answer returned an address but no `asn` is found.  How about that?